[2024-05-25 07:57:12,977] python INFO {'status': -1, 'error': {'message': "DatabaseAurora.get_data() missing 1 required positional argument: 'query'", 'function_name': 'get_data'}}
[2024-05-25 07:58:47,999] python INFO {'status': -1, 'error': {'message': 'syntax error at or near "table"\nLINE 1: SELECT * FROM table\n                      ^\n', 'function_name': 'get_data'}}
[2024-05-25 08:32:37,820] python INFO {'status': -1, 'error': {'message': 'syntax error at or near "/"\nLINE 1: ...YOB,cv_file_path,skills,experience,education,award/qualifica...\n                                                             ^\n', 'function_name': 'get_data'}}
[2024-05-25 08:32:56,114] python INFO {'status': -1, 'error': {'message': "DatabaseAurora.insert_many() missing 1 required positional argument: 'data'", 'function_name': 'insert_many'}}
[2024-05-25 08:34:00,167] python INFO {'status': -1, 'error': {'message': 'syntax error at or near "/"\nLINE 1: ...YOB,cv_file_path,skills,experience,education,award/qualifica...\n                                                             ^\n', 'function_name': 'insert_many'}}
[2024-05-25 08:34:48,127] python INFO {'status': -1, 'error': {'message': 'syntax error at or near "/"\nLINE 1: ...YOB,cv_file_path,skills,experience,education,award/qualifica...\n                                                             ^\n', 'function_name': 'insert_many'}}
[2024-05-25 08:37:07,418] python INFO {'status': -1, 'error': {'message': 'not all arguments converted during string formatting', 'function_name': 'insert_many'}}
[2024-05-25 08:37:39,471] python INFO {'status': -1, 'error': {'message': 'not all arguments converted during string formatting', 'function_name': 'insert_many'}}
[2024-05-25 08:38:06,445] python INFO {'status': -1, 'error': {'message': 'column "name" of relation "user_cv" does not exist\nLINE 1: INSERT INTO user_cv (id,name,age,cv_file_path, created_at, u...\n                                ^\n', 'function_name': 'insert_many'}}
